modules/ROOT/pages/backup-restore/modes.adoc

```diff
@@ -31,21 +32,29 @@ Doing full backup...
 ...
 ----
 ====
+
+The `neo4j-admin database backup` command with the `--type=full` argument creates a xref:backup-restore/online-backup.adoc#backup-artifact[full backup artifact] file in the format of _<databasename><timestamp>.backup_, called  in the target location (`--to-path`).
+
+The `neo4j-admin database backup` command allows you to include metadata, enabling the backup to store the role-based access control (RBAC) settings associated with the database.
+
 For more information about online backup options and how to control memory usage, see xref:backup-restore/online-backup.adoc[Back up an online database].
 
-[NOTE]
-====
-A full online database backup creates a xref:backup-restore/online-backup.adoc#backup-artifact[full backup artifact] in the target location (`--to-path`).
-====
+
+=== `neo4j-admin database dump`
+
+The `neo4j-admin database dump` command creates a single-file archive, that follows the format _<databasename><timestamp>.dump_, and stores it in the xref:configuration/file-locations.adoc#data[_<NEO4J_HOME>/data_] directory.
+
+Note that when using `neo4j-admin database dump`, only the database contents are backed up; database metadata and associated RBAC settings are not included.
```

Comment:
What are these RBAC settings? Also, I think we need to find all places where we mention `--include-metadata` and update them accordingly, as by default the backup command backs up all users and roles, unless you specified something different from the default `all`.

```suggestion
Note that the dump archive contains only the database without the associated users and roles metadata.
```

Reply:
I think this is in line with our RBAC -- https://neo4j.com/docs/operations-manual/current/authentication-authorization/manage-privileges/